awx_collection/tests/integration/targets/role/tasks/main.yml

```diff
@@ -55,9 +69,12 @@
         that:
           - "result is changed"
 
-    - name: Add Joe to the update role of the default Project with lookup Organization
+    - name: Add Joe and teams to the update role of the default Project with lookup Organization
       role:
         user: "{{ username }}"
```

Comment:
Should we still use `user` (and `team`) in tests if they're now deprecated?

Reply:
I left them in as while they are depreciated they should still be tested, however I added additional tests for the users feature.